.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -101,13 +101,13 @@ jobs:
           MAGMA_DEV_CPUS: 3
           MAGMA_DEV_MEMORY_MB: 9216
         working-directory: lte/gateway
-        # the image is directly concatenated to the registry, so we need the slash as delimiter
-        run: fab --show=debug --set DOCKER_REGISTRY=${{ inputs.registry }}/ integ_test_containerized:test_mode=${{ inputs.test_targets }}
+        run: |
+          fab -d integ-test-containerized --test-mode=${{ inputs.test_targets }} --docker-registry=${{ inputs.registry }}
```

Comment:
The removed comment sounds to me like in `${{ inputs.registry }}/` the slash is important - is this handled elsewhere? I don't see it in e.g., `_start_gateway_containerized`.

Reply:
tbh I don't really see why it is necessary, so far it has worked without it, but I can add that back in and see if it still works (then we can just keep it)

Reply:
I added the trailing slash, it seems to work